fobnail-attester.c

```diff
@@ -88,6 +93,216 @@ coap_context_t* att_coap_new_context(const bool enable_coap_block_mode)
 	return coap_context;
 }
 
+/* -------------------------------------------------------------- */
+
+#if 0
+
+	/* prepare primary key sensitive part */
+        TPM2B_AUTH authValuePK = {.size = 0, .buffer = {0}};
+        TPM2B_SENSITIVE_CREATE inSensitivePrimary = {.size = 0,
+                .sensitive = {
+                        .userAuth = authValuePK,
+                        .data = {.size = 0, .buffer = {0}},
+                }};
+
+/* prepare primary key public part */
+static TPM2B_PUBLIC primaryTemplate = {
+                .size = 0,
+                .publicArea =
+                        {
+                                .type = TPM2_ALG_RSA,
```

Comment:
FWIU this is for generation of AIK key, right? Can't we use ED25519 key? Trussed project which we use on Fobnail firmware doesn't support RSA.

Reply:
Need to investigate this question...
And why not RSA?